src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -277,8 +307,8 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         }
     });
 
-    fluid.slider.combineSliderOptions = function (sliderOptions, model, range) {
-        return $.extend(true, {}, sliderOptions, model, range);
+    fluid.slider.combineSliderOptions = function (sliderOptions, model, range, ariaOptions, strings) {
+        return $.extend(true, {}, sliderOptions, model, range, ariaOptions, strings);
```

Comment:
Is it safe to combine strings here? For example do we expect anyone to have collisions with names that they might  also have in strings.